Automattic/wp-calypso
# Add notice to let user resume copying site process

#### Proposed Changes

In this PR, I propose to add a Home page notice that will let users resume the unfinished copy site process.

#### Testing Instructions



1. Apply D99472-code
2. Navigate to `/sites`
3. Open the actions menu on a Business site.
4. Click on the Copy Site.
5. On the cart view, cancel the process to go back to the Home of the newly added site
6. Confirm the resume site copy notice in the UI.

#### Pre-merge Checklist





Related to 1516-gh-Automattic/dotcom-forge
